baklava/src/electron.ts

```diff
@@ -28,7 +28,7 @@ function createWindow() {
     mainWindow.webContents.openDevTools();
   }
   mainWindow.loadURL(
-    __prod__ ? `https://dogehouse.tv/` : "https://dogehouse.tv/"
+    __prod__ ? `https://dogehouse.tv/` : "http://localhost:3000/"
```

Comment:
Here

Reply:
no that's correct

Reply:
if `__prod__` is `true` choose the first URL otherwise choose the 2nd url